### DISCUSSION TITLE
.NET MAUI 8.0.91 (Service Release 9.1)

### DISCUSSION THREAD

**Discussion by davidortinau** [dotnet/maui#24947] [Announcements]
This release is now available via [NuGet.org](https://www.nuget.org/packages/Microsoft.Maui.Controls/8.0.91) and is compatible with the latest .NET 8 workload. 

```
<ItemGroup>
  <PackageReference Include="Microsoft.Maui.Controls" Version="8.0.91" />
  <PackageReference Include="Microsoft.Maui.Controls.Compatibility" Version="8.0.91" />
</ItemGroup>
```

The focus of our service releases is resolving high-impact bugs for developers upgrading their applications from Xamarin to .NET 8.

- [Release notes](https://github.com/dotnet/maui/releases/tag/8.0.91)

**Comment by tschbc** [comment]
I appreciate the more frequent releases! It's been really nice to see.

**Reply by TedMobile** [reply]
Absolutely. More frequent releases with fixes is excellent

**Reply by follesoe** [reply]
Completely agreed. Good to see the cadence and amount of fixes being rolled out!

**Reply by ronanj2** [reply]
David and team, keep up the great work!

**Comment by HuaFangYun** [comment]
I feel that I need to fix some general problems and performance faster and keep Bug fixes above 30%. That would be better. Thank you for providing this platform, but it still needs to be improved and optimized harder to get better.
